internal/daemon/controller/handlers/hosts/host_service.go

```diff
@@ -601,11 +601,11 @@ func toProto(ctx context.Context, in host.Host, opt ...handlers.Option) (*pb.Hos
 
 // A validateX method should exist for each method above.  These methods do not make calls to any backing service but enforce
 // requirements on the structure of the request.  They verify that:
-//  * The path passed in is correctly formatted
-//  * All required parameters are set
-//  * There are no conflicting parameters provided
-//  * The type asserted by the ID and/or field is known
-//  * If relevant, the type derived from the id prefix matches what is claimed by the type field
+//   - The path passed in is correctly formatted
```

Comment:
I see someone has upgraded to 1.19 😁

Reply:
We should probably merge https://github.com/hashicorp/boundary/pull/2347 sooner than later to get some more cycles

Reply:
Accidentally, I forgot to exclude it from package updates.

Reply:
But if you want to take it as tacit approval to push through your PR to upgrade main to 1.19, I'm on board :-)